0.9.0 make error POPT_ARG_ARGV ? POPT_ARG_VAL

tools/publish.c:87:23: error: 'POPT_ARG_ARGV' undeclared (first use in this function)
       {"header", 'H', POPT_ARG_ARGV, &headers, 0,

/usr/local/rabbitmq-c/tools/publish.c:87: error: ‘POPT_ARG_ARGV’ undeclared (first use in this function)
/usr/local/rabbitmq-c/tools/publish.c:87: error: (Each undeclared identifier is reported only once
/usr/local/rabbitmq-c/tools/publish.c:87: error: for each function it appears in.)
gmake[2]: *** [tools/CMakeFiles/amqp-publish.dir/publish.c.o] Error 1


Same error

version：v0.9.0

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -167,7 +167,7 @@ if(BUILD_EXAMPLES)
 endif()
 
 if(BUILD_TOOLS)
-  find_package(POPT REQUIRED)
+  find_package(POPT 1.14 REQUIRED)
   if(BUILD_TOOLS_DOCS)
     find_package(XMLTO REQUIRED)
   endif()

--- a/cmake/FindPOPT.cmake
+++ b/cmake/FindPOPT.cmake
@@ -4,6 +4,12 @@
 #  POPT_FOUND - System has popt
 #  POPT_INCLUDE_DIR - The popt include directory
 #  POPT_LIBRARY - The libraries needed to use popt
+#  POPT_VERSION - The version of popt that was found
+#
+# This module also supports version requirements, e.g.
+# find_package(POPT 1.14 REQUIRED). popt does not expose its release
+# version in popt.h, so the version is obtained from pkg-config. A version
+# requirement therefore needs pkg-config and the popt.pc file to be available.
 
 # use pkg-config to get the directories and then use these values
 # in the FIND_PATH() and FIND_LIBRARY() calls

